style.css

```diff
@@ -1673,7 +1676,7 @@ textarea {
   backdrop-filter: blur(5px);
 }
 
-#modal {
+.modal {
   max-width: 80%;
   height: 90%;
```

Comment:
- [ ] As the prev reviewer asked, watch out for the spacing, and font color in your popup window to match the given design. I marked the issues I found below, 
![image](https://user-images.githubusercontent.com/91486791/232090695-4d72ff69-e0a3-4e7e-9cf1-e85d4497f2c3.png)

- [ ] Please set the color for the project description to the exact color of your title as in the figma design
- [ ] Please make your project links colored purple
- [ ] In mobile screens, your text overflows the screen. Kindly fix this.
![image](https://user-images.githubusercontent.com/91486791/232090858-8804f079-a362-4eb0-a094-c69ba58d4d50.png)

- In the popup window, kindly set overflow to hidden to prevent scrolling on the main page.
- I also noticed, in your dynamically created project section, the container hasn't been set correctly to the center horizontally. Kindly fix that too as it might occur after you delete the work section from HTML.
![image](https://user-images.githubusercontent.com/91486791/232092486-a5f01184-f123-4ae1-945b-6ded6361dfe4.png)
- [ ] Kindly arrange the spacing around this sections as it doesn't seem as in the figma design:
![image](https://user-images.githubusercontent.com/91486791/232092905-e7182d28-38cf-452e-9867-adb81a80de95.png)

Reply:
Well Thank you for the Detailed Review! I am doing my best to make it look like actual.